bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/protocol/xml/InputWithPresetControlXML.java

```diff
@@ -156,7 +154,7 @@ private int convertToPresetNumber(String presetValue) {
                     if (Character.isLetter(presetAlpha) && Character.isUpperCase(presetAlpha)
                             && Character.isDigit(presetValue.charAt(1))) {
                         int presetNumber = Integer.parseInt(presetValue.substring(1));
-                        return (ArrayUtils.indexOf(LETTERS, presetAlpha) + 1) * 100 + presetNumber;
+                        return (new String(LETTERS).indexOf(presetAlpha) + 1) * 100 + presetNumber;
```

Comment:
You could probably prevent some garbage collection by turning `LETTERS` into a String itself.
Then you can use `char presetAlpha = LETTERS.charAt(..)` below.

Reply:
Good idea. I have made the change. Thanks